app/forms/validators.py

```diff
@@ -48,11 +49,17 @@ def __call__(
         form: FlaskForm,
         field: Union[DecimalFieldWithSeparator, IntegerFieldWithSeparator],
     ) -> None:
+        try:
+            if math.isnan(float(field.raw_data[0])):
+                raise validators.StopValidation(error_messages["INVALID_NUMBER"])
```

Comment:
Not familiar with wtforms validation - would be good to have a quick chat about this raise-pass pattern so I can understand it.

Reply:
I mean, it is more general logic than wtforms, here we only want to raise an error (stop validation and raise error) when `isnan()` can be executed, some of the data we try to perform `isnan()` on will be invalid input and will break runner.  But I don't want to break runner on inputs that I don't want to check. That's why I had to add `except` type and value errors, for all the edge cases. But when inputs are not valid I `pass` cause I want to continue to the next lines. I only want to make this nan check on numeric inputs otherwise do the rest of the validation.